enterprise/server/cmd/ci_runner/main.go

```diff
@@ -676,6 +676,21 @@ func run() error {
 			return status.WrapError(err, "failed to extract bazelisk")
 		}
 		*bazelCommand = bazeliskPath
+
+	}
+
+	// Symlink bazel to `bazelCommand` so that it can be easily invoked
+	if err := os.Symlink(*bazelCommand, filepath.Join(rootDir, bazelBinaryName)); err != nil {
+		if !strings.Contains(strings.ToLower(err.Error()), "file exists") {
```

Comment:
nit: it'd be slightly more robust to remove any existing symlink first - then you don't have to check for "file exists" and you also have a stronger guarantee that the resulting symlink has the correct target in the case where it already exists

```
bazelPath := filepath.Join(rootDir, bazelBinaryName)
if err := os.RemoveAll(bazelPath); err != nil {
  return status.WrapError(err, "remove existing bazel binary")
}
if err := os.Symlink(*bazelCommand, bazelPath); err != nil {
  return status.WrapError(err, "symlink bazel to bazelisk")
}
```